etc/bin/run-validate-tutorials

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+set -eu
+cd "$(dirname "$0")"
+
+for FILE in $(echo ../cylc/flow/etc/tutorial/*/.validate) ; do
```

Comment:
This path looks like it needs updating
```
running ../cylc/flow/etc/tutorial/*/.validate
etc/bin/run-validate-tutorials: line 25: cd: ../cylc/flow/etc/tutorial/*: No such file or directory
```
https://github.com/cylc/cylc-flow/runs/4799870817?check_suite_focus=true#step:7:8